browsers/opera_android.json

```
@@ -314,6 +314,13 @@
           "status": "current",
```

Just a quick little fix, and I'd say this looks good!
```suggestion
          "status": "retired",
```